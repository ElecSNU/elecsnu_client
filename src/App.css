#app,
body,
#root,
html {
    height: 100%;
}

#app {
    display: flex;
    flex-direction: column;
}

#main-page {
    height: 92.5%;
    display: grid;
    place-items: center;
}
