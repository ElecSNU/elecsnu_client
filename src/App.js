import React, { Suspense } from 'react';
import { ToastContainer } from 'react-toastify';

import 'react-toastify/dist/ReactToastify.css';
import './App.css';

const Navbar = React.lazy(() =>
    import('./components/Navbar/index')
);
const MainPage = React.lazy(() => import('./MainPage'));

function App() {
    return (
        <Suspense fallback={<div></div>}>
            <div id='app'>
                <ToastContainer />
                <Navbar />
                <MainPage />
            </div>
        </Suspense>
    );
}

export default App;
