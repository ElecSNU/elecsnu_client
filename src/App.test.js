import { render, screen } from '@testing-library/react';
import App from './App';
import renderer from 'react-test-renderer'

test('checks the working of the website', () => {
    const div = document.createElement("div")
    const tree = renderer.create(<index/>).toJSON();
    expect(tree).toMatchSnapshot();
});
