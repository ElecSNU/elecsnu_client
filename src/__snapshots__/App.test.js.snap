// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`checks the working of the website 1`] = `<index />`;
